src/wurf/git.py

```diff
@@ -22,7 +23,7 @@ def cmd_and_log(self, *args, **kwargs):
         :param kwargs: Keyword arguments passed to ctx.cmd_and_log(...)
         """
         args = [self.git_binary] + list(args)
-        return cmd_and_log(args, **kwargs)
+        return self.ctx.cmd_and_log(args, **kwargs)
```

Comment:
before ``cmd_and_log`` was not defined. ``self.ctx.cmd_and_log`` is used in all the other functions in this file so I assumed that it was that ``self.ctx.cmd_and_log`` should be used here as well.